### ISSUE TITLE
Exit wound Debuff (234) on 5 charges isn't effecting parry

### ISSUE THREAD

**Issue by Twinkybell** [WarEmu/WarBugs#10519]
We have tested out Exit Wound Debuff (234) on 5 charges is not effecting parry.In Getstats and in Charecter tab.Seeker's blade effects Charecter Tab and Getstats but Exit Wound debuff does not and it should do.

![elerreanns_024](https://user-images.githubusercontent.com/32363217/30969052-2bf12482-a459-11e7-8285-96a8297af0da.jpg)
![elerreanns_023](https://user-images.githubusercontent.com/32363217/30969053-2bf527bc-a459-11e7-889e-5013fde2e685.jpg)
![elerreanns_022](https://user-images.githubusercontent.com/32363217/30969049-2beddae8-a459-11e7-9b7f-d69b364e5219.jpg)
![elerreanns_021](https://user-images.githubusercontent.com/32363217/30969051-2bf040d0-a459-11e7-953d-59268158d83f.jpg)
![elerreanns_020](https://user-images.githubusercontent.com/32363217/30969050-2bee720a-a459-11e7-87b3-27248a042b75.jpg)

**Comment by tionblack** [comment]
Yes please fix this issue , its a top tier talent  witch hunter has  the stat debuff for weapon skill is not working as intented in relation to weapon skill to parry  bonus.
